src/main/java/io/lettuce/core/ClientOptions.java

```diff
@@ -194,6 +216,60 @@ public Builder autoReconnect(boolean autoReconnect) {
             return this;
         }
 
+        /**
+         * Enables or disables periodic reauthenticate of active connections. Defaults to {@code false}. See
+         * {@link #DEFAULT_AUTO_REAUTHENTICATE}.
+         *
+         * @param periodicReauthenticate true/false
+         * @return {@code this}
+         * @since 6.2
+         */
+        public Builder enablePeriodicReauthentication(boolean periodicReauthentication) {
+            this.shouldReauthenticatePeriodically = periodicReauthentication;
+            return this;
+        }
+
+        /**
+         * Enables periodic reauthenticate of active connections. Defaults to {@code false}. See
+         * {@link #DEFAULT_AUTO_REAUTHENTICATE}.
+         *
+         * @return {@code this}
+         * @since 6.2
+         */
+        public Builder enablePeriodicReauthentication() {
+            return enablePeriodicReauthentication(true);
+        }
+
+        /**
+         * Enables periodic reauthenticate and sets the reauthentication period. Defaults to {@literal 60 MINUTES}. See
+         * {@link #DEFAULT_REAUTHENTICATE_PERIOD} and {@link #DEFAULT_REAUTHENTICATE_PERIOD_UNIT}. This method is a shortcut for
+         * {@link #reauthenticationPeriod(Duration)} and {@link #enablePeriodicReauthentication()}.
+         *
+         * @param reauthenticationPeriod period for triggering reauthentication, must be greater {@literal 0}
+         * @return {@code this}
+         * @since 6.2
+         */
+        public Builder enablePeriodicReauthentication(Duration reauthenticationPeriod) {
+            return reauthenticationPeriod(reauthenticationPeriod).enablePeriodicReauthentication();
+        }
+
+        /**
+         * Set the refresh period. Defaults to {@literal 60 MINUTES}. See {@link #DEFAULT_REAUTHENTICATE_PERIOD} and
```

Comment:
refresh -> reauthentication
because you don't use the term `refresh` elsewhere for this.